CONTRIBUTING.md

```diff
@@ -13,10 +13,7 @@ Follow these steps to build from source and deploy:
 
 1. `docker build --rm -f "Dockerfile" -t <repo>/kubecost-cost-model:<tag> .`
 2. Edit the [pulled image](https://github.com/kubecost/cost-model/blob/master/kubernetes/deployment.yaml#L25) in the deployment.yaml to <repo>/kubecost-cost-model:<tag>
-3. Set [this environment variable](https://github.com/kubecost/cost-model/blob/master/kubernetes/deployment.yaml#L33) to the address of your prometheus server
-4. `kubectl create namespace cost-model`
```

Comment:
Let's touch base before we launch markdown changes!

Reply:
Definitely. I don't want to break links from elsewhere especially.

I think the changes I've made make the flow clearer (in my head at least). I think https://github.com/kubecost/cost-model/blob/mmd-exporter-only/INSTALL.md looks better than the scattering of info between `deploying-as-a-pod.md`, `CONTRIBUTING.md`, etc.

Reply:
Backed off the md changes in light of our discussion.